source/Player.hx

```diff
@@ -133,4 +146,16 @@ import flixel.input.keyboard.FlxKey;
   {
     return FlxG.keys.anyPressed([FlxKey.Z]);
   }
+
+  private function updateHitBoxes():Void
+  {
+    var hitBoxHeight = 3;
+    var hitBoxOffset = 1;
+    var hitBoxWidth = width - hitBoxOffset * 2;
+    var hitBoxX = x + hitBoxOffset;
+
+    topBox.x = btmBox.x = hitBoxX;
+    topBox.y = y + hitBoxOffset;
+    btmBox.y = y + height - hitBoxOffset - hitBoxHeight;
```

Comment:
By this math, these hitboxes are inside the player's main hitbox. 
I don't think this is the desired behavior, because it could lead to instances where the player overlaps with an enemy below but the bottom hitbox does not.